lib/signs/bus.ex

```diff
@@ -340,6 +393,23 @@ defmodule Signs.Bus do
     period.(current_time) != period.(last_read_time)
   end
 
+  # In addition to periodic audio messages, we also read the drawbridge message by itself if all
+  # of the following are true:
+  # 1. the drawbridge just went up
+  # 2. drawbridge messages are enabled
+  # 3. we are at a stop that is impacted, but does not show visual drawbridge messages
```

Comment:
Clarifying question: how is this 3rd condition expressed in the logic below?

Reply:
It's `chelsea_bridge == "audio"`. When `chelsea_bridge == "audio_visual"`, we append a visual message to the sign (and don't trigger this immediate readout). There's a TWE comment that explains the rationale, I'll add that for clarity.